File: MyBreak/DayTwo/AllOver/JavaScript/Chapter4TaskOne.js
const prompt = require("prompt-sync")()

let counter = 0
let total = 0
while(grades !=  -1){
  total += grades
  counter += 1
  let grades = prompt("Enter your student grades or -1 to exit ")
}
if(average != 0){
  average = total / grades
  console.log("The total of "  + counter + " grade  is " + counter + grades)
  console.log("The class average is " + average);
}
else{
  console.log("no grades were received")
}

File: MyBreak/DayTwo/AllOver/Python/chapter4_task_one.py
counter = 0
total = 0

while grades != -1:
  total += grades
  counter += 1
  grades = int(input("Enter student grades or -1 to exit the program"))
  
if average != 0:
  average = total / grades
  print("The total of " + str(counter) + "student is " + str(total), counter)
  print("the class average is ",  average)
  
else:
  print("no grades were received")